reference/SshBasics/main.cpp

```diff
@@ -58,8 +58,11 @@ CONSOLE_APP_MAIN
 		ForwardTcpIp(session);
 #elif defined(SSH_PICK_SEMANTICS)
 		SshPick(session);
+#elif defined(SSH_POLYMORPHISM)
```

Comment:
Where is SSH_POLYMORPHISM flag defined?

Reply:
> Where is SSH_POLYMORPHISM flag defined?

Users *can* define it if they would like to run the specific example. By default the `SSH_SFTP` flag is defined. (this has been the case since the beginning.)